library-extensions/src/main/java/com/mikepenz/fastadapter_extensions/utilities/SubItemUtil.java

```diff
@@ -305,6 +305,54 @@ else if (!subItemsOnly && getParent(item) == null) {
         return deleted;
     }
 
+    /**
+     * notifies sub items, if they are currently extended
+     *
+     * @param adapter the adapter
+     * @param identifiers set of identifiers that should be notified
+     */
+    public static <T extends IItem & IExpandable> void notifyItemsChanged(final FastAdapter adapter, Set<Long> identifiers) {
```

Comment:
Replace T with item, because it looks better. Replace FastAdapter with FastAdapter<Item> so you don´t need the cast anymore.